### DISCUSSION TITLE
Need Easier Access to Filter in Data Explorer

### DISCUSSION THREAD

**Discussion by HaroldDMiller** [posit-dev/positron#4334] [Feedback]
I really like Positron, but there is one thing that is keeping me from using it --  it is too hard to do a simple filter/search in the Data Explorer pane.  In RStudio, if you click on the Filter button, a field appears under every column heading where you can simply type in something to find the rows that contain what you typed (you don't even have to hit enter -- it immediately starts filtering as you type).  In VSCode, if you click on the hamburger menu icon in the column name, it gives you an empty field and filter that is defaulted to "Contains," so you can immediately type something in and filter to the rows that contain what you typed.  But in Positron, you have click Add Filter for the column, and then you have to select what filter you want (it does not default to "Contains") and only then can you actually type in the filter.  It takes a lot of clicks in Positron to do what takes 1 or 2 clicks in RStudio or VSCode, and if you're trying to find rows based on column values multiple times, it is a very slow process.  I do a lot of lookups both to find specific records and to find the reasons for anomalies in calculations, etc., so right now, Positron is a much less productive app for me than either VSCode or RStudio.  Fixing this would make me start using it instead of VSCode.

**Comment by jmcphers** [comment]
Thanks for this feedback! We've got a number of issues tracking improvements to the efficiency of adding filters in the Explorer -- feel free to upvote/chime in on these:

https://github.com/posit-dev/positron/issues/3302
https://github.com/posit-dev/positron/issues/3772
https://github.com/posit-dev/positron/issues/2774
